### PR TITLE
Scan images with trivy

### PR DESCRIPTION
**Pull request by janisz**
Scan latest images of main, roxctl and central-db with trivy for critical vulnerabilities. 

1 issue found https://github.com/stackrox/stackrox/security/code-scanning/53 fixed in https://github.com/stackrox/stackrox/pull/1994

Refs: #1672

### CODE REVIEW

File: .github/workflows/trivy.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        image:
+          - main
+          - roxctl
+          - central-db
+    runs-on: ubuntu-18.04
+    steps:
+          - name: Run Trivy vulnerability scanner in repo mode
+            uses: aquasecurity/trivy-action@master
+            with:
+              image-ref: 'quay.io/stackrox-io/${{ matrix.image }}:latest'
```
Comment (vikin91) [inline .github/workflows/trivy.yml:30]: It looks good in general, but ca we be sure that the `latest` will always correspond to the image built in the current PR?

**Review by vikin91 — APPROVED** [review]

**Review by janisz — COMMENTED** [review]
It's not meant to run on every PR. There is cron schedule to do it nightly.

**Comment by vikin91** [comment]
Just out of curiosity: what have you decided instead of this?

**Comment by msugakov** [comment]
Context: https://srox.slack.com/archives/CELUQKESC/p1652969893141889?thread_ts=1652788176.176129&cid=CELUQKESC

**Pull request closed without merging**